site/features.md

```diff
@@ -53,6 +54,29 @@ now re-use the same tools and techniques when deploying applications
 as containers as we would have done when deploying them 'on metal' in
 our data centre.
 
+### <a name="ipam"></a>IP Address Management
+
+Weave can assign unique IP addresses to each container across the
+network.  To make this work, weave must be told what range of
+addresses to allocate from, on each host.  For example:
```

Comment:
PR #552 will bring a mismatched allocator range out to `weave status` - it will break the connection and show up in the 'reconnects' section